apps/web/src/pages/templates/components/templates-store/TemplatesStoreModal.tsx

```diff
@@ -102,6 +103,19 @@ export const TemplatesStoreModal = ({ general, isOpened, onClose }: ITemplatesSt
     >
       <ModalBodyHolder>
         <TemplatesSidebarHolder>
+          {popular.map((group) => (
+            <TemplatesGroup key={group.name}>
+              <GroupName>{group.name}</GroupName>
+              {group.blueprints.map((template) => {
+                return (
+                  <TemplateItem key={template.name} onClick={() => handleTemplateClick(template)}>
+                    <FontAwesomeIcon icon={template.iconName} />
+                    <span>{template.name}</span>
+                  </TemplateItem>
+                );
+              })}
+            </TemplatesGroup>
+          ))}
```

Comment:
forgot to show the popular blueprints the the Templates Store modal, they will be rendered at the top